src/icalendar/attr.py

```diff
@@ -644,7 +652,11 @@ def _del_categories(component: Component) -> None:
 
 .. note::
 
-   At present, we do not take the LANGUAGE parameter into account.
+    At present, we do not take the LANGUAGE parameter into account.
+
+.. seealso::
+
+    :attr:`Component.concepts`
```

Comment:
This doesn't render, and I don't know why. See https://icalendar.readthedocs.io/en/latest/reference/api/icalendar.html#icalendar.Availability.categories and https://icalendar.readthedocs.io/en/latest/reference/api/icalendar.cal.component.html. Is this even a thing?

```suggestion
.. seealso::

    :py:attr:`Component.concepts`
```